orchestrator-salt/src/main/resources/salt-common/salt/telemetry/init.sls

```diff
@@ -1,12 +1,12 @@
 {%- from 'telemetry/settings.sls' import telemetry with context %}
-{% if telemetry.databusEndpointValidation and telemetry.databusEndpoint and telemetry.cdpTelemetryVersion > 1 %}
+{% if telemetry.databusEndpointValidation and telemetry.databusEndpoint %}
 check_databus_network_connectivity:
   cmd.run:
-    - name: "cdp-telemetry utils check-connection --url {{ telemetry.databusEndpoint }}"
+    - name: "curl {{ telemetry.databusCurlConnectOpts }} -s -k {{ telemetry.databusEndpoint }} > /dev/null"
     - failhard: True{% if telemetry.proxyUrl %}
```

Comment:
coudl you move these if and endif stuff to new line, it's pretty hard to follow everything in one line, indentation should help code readability